### PR TITLE
Addition of boss banner

### PR DESCRIPTION
**Pull request by 0tuedon**
Adds the Boss Banner

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -103,39 +103,41 @@ export const App = () => {
...
   return (
-    <div
-      className={`${
-        isHomePage ? "relative" : ""
-      } bg-custom-background text-custom-primary-text`}
-    >
-      <main
-        id="main"
-        className="min-h-[95dvh] flex w-full items-center pt-[60px] md:pt-[76px] 2xl:pt-[122px]"
+    <>
```
Comment (Emmanuel-Develops) [inline src/pages/index.tsx:106]: remove the react fragment, there should be no changes on this file related to adding boss banner

**Review by Emmanuel-Develops — CHANGES_REQUESTED** [review]

**Comment by Emmanuel-Develops** [comment]
On the smallest of screens, the height may not be enough to fit its content.
Screenshot 2024-11-28 at 8 47 40 PM

use the styles prop to change the height, it is a mistake from bdp-ui.
```
<Banner
        headingText="Start your career in bitcoin open source —"
        linkText="APPLY TODAY"
        linkTo="/"
        hasBoss
        styles={{
          container: "h-[auto_!important] py-2"
        }}
      />
```

**Comment by 0tuedon** [comment]
> styles={{
>           container: "h-[auto_!important] py-2"
>         }}

Thanks

**Review by Emmanuel-Develops — APPROVED** [review]

**Comment by Emmanuel-Develops** [comment]
> Thanks

That was a preemptive suggestion, we do animate height so this overwrites what we have and breaks the close sequence

**Pull request merged**